lib/ansible/modules/cloud/digital_ocean/digital_ocean_floating_ip.py

```diff
@@ -269,20 +269,39 @@ def associate_floating_ips(module, rest):
 def create_floating_ips(module, rest):
     payload = {
     }
+    floating_ip_data = None
 
     if module.params['region'] is not None:
         payload["region"] = module.params['region']
+
     if module.params['droplet_id'] is not None:
         payload["droplet_id"] = module.params['droplet_id']
 
-    response = rest.post("floating_ips", data=payload)
-    status_code = response.status_code
-    json_data = response.json
-    if status_code == 202:
-        module.exit_json(changed=True, data=json_data)
+        page = 1
```

Comment:
Can you please use ? -https://github.com/ansible/ansible/blob/f21f3219b2a925a87e0c8aa54eb8ceca2cb55dc5/lib/ansible/module_utils/digital_ocean.py#L114

Reply:
@Akasurde done